java/src/jmri/jmrix/loconet/LnSensor.java

```diff
@@ -45,7 +45,7 @@ private void init(String systemName, String prefix) {
         }
 
         // At construction, register for messages
-        tc.addLocoNetListener(~0, this);
+        //tc.addLocoNetListener(~0, this);
```

Comment:
Why is this commented out here, but removed in every other class?

Reply:
Dont know why, just removed it.